drivers/rs485_seds/rs485_sedsconfiguration.asn

```diff
@@ -0,0 +1,31 @@
+-- DRIVER CONFIGURATION MODEL [user-editable]
+-- Shall contain definition of Driver Configuration.
+-- This file will be used by ocarina-components.aadl and Board1.xml.
+RS485-SEDS-DRIVER-CONFIGURATION DEFINITIONS ::= BEGIN
+
+    RS485-SEDS-Device-T    ::= ENUMERATED {uart0, uart1, uart2, uart3, uart4}
+
+    RS485-SEDS-Baudrate-T  ::= ENUMERATED {b9600, b115200}
+
+    RS485-SEDS-PioHwas-Port-T ::= ENUMERATED {
+        pioHwas-Port-A(0),
+        pioHwas-Port-B(1),
+        pioHwas-Port-C(2),
+        pioHwas-Port-D(3),
+        pioHwas-Port-E(4)
+    }
+
+    RS485-SEDS-PioHwas-Pin-T ::= INTEGER(0 .. 31)
+
+    RS485-SEDS-PioHwas-T ::= SEQUENCE {
+        mPortConfig RS485-SEDS-PioHwas-Port-T,
```

Comment:
Rather port ID and pin ID/number; pinconfig has a different meaning (pullup, peripheral, input, schmidt....(

Reply:
done